samples/snippets/process_document_quality_sample.py

```diff
@@ -15,64 +15,83 @@
 
 # [START documentai_process_quality_document]
 
+from typing import Sequence
+
+from google.api_core.client_options import ClientOptions
+from google.cloud import documentai_v1 as documentai
+
 # TODO(developer): Uncomment these variables before running the sample.
-# project_id= 'YOUR_PROJECT_ID'
-# location = 'YOUR_PROJECT_LOCATION' # Format is 'us' or 'eu'
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_PROCESSOR_LOCATION' # Format is 'us' or 'eu'
 # processor_id = 'YOUR_PROCESSOR_ID' # Create processor in Cloud Console
 # file_path = '/path/to/local/pdf'
+# mime_type = 'application/pdf' # Refer to https://cloud.google.com/document-ai/docs/processors-list for supported file types
 
 
 def process_document_quality_sample(
```

Comment:
Same question about splitting this function into two

Reply:
Similar answer, this example is focused on handling the processing response, not making the request